### PR TITLE
owner(cdc): add barrier ts metrics monitoring

### LINKED ISSUE #4821: changefeed  barrierTs monitoring

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #4821 

### What is changed and how it works?
* add `barrierTs` to `Changefeed checkpoint` panel, which is useful to trace table scheduling.
* split `resolved ts` and `checkpoint` related panels. add `processor` metrics to it.
image
* bump Grafana to 7.5, and use new `Table` panel, `Ownership` / `Changefeed table count` / `Table count maintained by Owner` / `Schema Storage GC progress` / `Stream Count Between Nodes`
* remove `Processor resolved ts` table, since it's record in `Changefeed resolved ts` panel.
image

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### Release note 

```release-note
`None`
```

### CODE REVIEW

**Review by sdojjy — APPROVED** [review]

**Comment by overvenus** [comment]
Could you convert schema storage GC progress to the graph like changefeed resolved ts, I think it's more informative. 

Also, let's put checkpoint ts first, I think users care checkpoint ts more than resolved ts, eg:

|  |  |
|-|-|
| image | image |

**Comment by 3AceShowHand** [comment]
> Could you convert schema storage GC progress to the graph like changefeed resolved ts, I think it's more informative.
> 
> Also, let's put checkpoint ts first, I think users care checkpoint ts more than resolved ts, eg:
> 
> image	image

how about this.

image

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -17655,6 +18823,6 @@
   },
   "timezone": "browser",
   "title": "Test-Cluster-TiCDC",
-  "uid": "YiGL8hBZ1",
-  "version": 37
-}
+  "uid": "YiGL8hBZ1x",
```
Comment (overvenus) [inline metrics/grafana/ticdc.json:18826]: Please do not change uid.

Comment (3AceShowHand) [inline metrics/grafana/ticdc.json:18826]:
```suggestion
  "uid": "YiGL8hBZ1",
```

**Review by overvenus — APPROVED** [review]

**Pull request merged**